spec/javascripts/analytics.spec.js

```diff
@@ -0,0 +1,416 @@
+/* eslint no-prototype-builtins: "error" */
```

Comment:
When I run the tests locally, this entire spec file is skipped?

Reply:
It should be skipped because it is using `xdescribe`.